gendiff/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .scripts.gendiff import generate_diff
```

Comment:
wdyt about running `mypy` in strict mode? it will provide more checks (and errors too, probably :))